objects/src/assets/mod.rs

```diff
@@ -57,6 +57,11 @@ impl Asset {
             _ => false,
         }
     }
+
+    /// Returns true if this asset is a fungible asset.
+    pub fn is_fungible(&self) -> bool {
```

Comment:
optional nit: this function can be `const fn`

Reply:
Fixed!